shared/Cargo.toml

```diff
@@ -8,5 +8,6 @@ anyhow      = "^1"
 crossterm   = "^0"
 libc        = "^0"
 parking_lot = "^0"
-ratatui  		 = { version = "^0" }
+rand        = "^0"
```

Comment:
How about using the md5 with the current time? That way don't need to introduce an extra crate

Reply:
Simply use the timestamp, no need for md-5.